### PR TITLE
Basic breadcrumb for PLC

### PR DESCRIPTION
**Pull request by mehalshah**
Basic breadcrumb for plc, also make the course view take users to the unit view when they click on unit titles

image

### CODE REVIEW

**Comment by aoby** [comment]
looks unrelated, but circleCI is failing.

**Comment by mehalshah** [comment]
No - the Circle failure is my fault

File: code-studio/src/js/plc/header.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    };
+
+    let breadcrumbIconStyle = {
+      margin: 5,
+      color: color.charcoal
+    };
+
+    let currentPageStyle = {
+      color: color.orange
+    };
```
Comment (aoby) [inline code-studio/src/js/plc/header.jsx:26]: Our [React style guide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react) says to put all css styles in single `styles` object.

**Comment by aoby** [comment]
lgtm, after style change and once tests pass

File: code-studio/src/css/plc/teacher_dashboard.scss [hunk 2]
```diff
@@ -42,8 +42,12 @@
     width: 400px;
 
     .course_unit_title {
+      a {
+        color: $dark_charcoal !important;
```
Comment (joshlory) [inline code-studio/src/css/plc/teacher_dashboard.scss:46]: Why `!important`?  Can we increase specificity instead?

File: code-studio/src/js/plc/header.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+var PlcHeader = React.createClass({
+  propTypes: {
+    unit_name: React.PropTypes.string,
+    course_view_path: React.PropTypes.string
+  },
+
+  render: function () {
+    let breadcrumbStyle = {
+      container: {
+        marginBottom: '20'
```
Comment (joshlory) [inline code-studio/src/js/plc/header.jsx:17]: Nit: don't need quotes.

File: dashboard/app/views/scripts/show.html.haml [hunk 4]
```diff
@@ -1,7 +1,18 @@
...
   #lesson
-    - if !@script.professional_course?
+    - if @script.professional_learning_course?
+      #breadcrumb
+      - react_props = {unit_name: @script.plc_course_unit.unit_name, course_view_path: professional_learning_path(@script.plc_course_unit.plc_course.name)}
+      :javascript
+        ReactDOM.render(
+          React.createElement(window.dashboard.plcHeader, #{react_props.to_json}),
+          document.getElementById('breadcrumb')
+        );
```
Comment (joshlory) [inline dashboard/app/views/scripts/show.html.haml:14]: This should really live in the `code-studio` bundle.

File: code-studio/src/js/plc/header.jsx (hunk 3 above)
Comment (joshlory) [inline code-studio/src/js/plc/header.jsx:17]:
```
Warning: a `div` tag (owner: `unknown`) was passed a numeric string value for CSS property `marginBottom` (value: `20`) which will be treated as a unitless number in a future version of React.
```

**Pull request merged**